packages/console/src/cloud/pages/Main/index.tsx

```diff
@@ -46,18 +46,14 @@ function Protected() {
   }
 
   if (tenants) {
-    if (currentTenantId) {
-      return <Redirect tenants={tenants} toTenantId={currentTenantId} />;
+    if (tenants.length > 0) {
+      /**
+       * Redirect to the first tenant if the current tenant ID is not set or can not be found.
+       */
+      return <Redirect tenants={tenants} toTenantId={currentTenantId || tenants[0].id} />;
```

Comment:
nit: use `??`

Reply:
If the `currentTenantId` is 0, then it would fallback if you use `??`. So IMO `||` is fine here